src/index.html

```diff
@@ -7,12 +7,65 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,900&display=swap"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The link to the stylesheet should have an `href` attribute with a `.css` file instead of `.scss`. While SCSS files are used during development, they need to be compiled to CSS for the browser to understand them.